### PR TITLE
September maintenance list

### PR DESCRIPTION
**Pull request by i0natan**

### CODE REVIEW

**Comment by i0natan** [comment]
@sagirk Isn't it about time to add you to the core team list?

**Comment by sagirk** [comment]
@i0natan Haha yeah... I'm behind on a bunch of things, including this! 😅

File: README.md [hunk 1]
```diff
@@ -1053,6 +1053,10 @@ This repository is being kept up to date thanks to the help from the community.
 🌻 [Gabe Kuslansky](https://github.com/GabeKuslansky),
 🌻 [Ron Gross](https://github.com/ripper234),
 🌻 [Valeri Karpov](https://github.com/vkarpov15)
+🌻 [Sergio](https://github.com/imsergiobernal),
+🌻 [Duarte Mendes](https://github.com/duartemendes),
+🌻 [Nikola Telkedzhiev](https://github.com/ntelkedzhiev),
+🌻 [Vitor Godoy](https://github.com/vitordagamagodoy)
```
Comment (sagirk) [inline README.md]: Seems like we missed awarding a flower to @manishsaraan for contributing https://github.com/i0natan/nodebestpractices/pull/241

Comment (sagirk) [inline README.md]: It appears that we also forgot to add @uronly14me for contributing https://github.com/i0natan/nodebestpractices/pull/236 in August.

**Comment by i0natan** [comment]
@sagirk Yes, of course, add your BIO and PR. 

Great that you paid attention to the credits we forgot

**Comment by js-kyle** [comment]
I've added the 2x missing contributors from September, so lets merge this one? Then a bio from @sagirk when you find the time

**Pull request merged**